api/chat.go

```diff
@@ -15,13 +15,15 @@ import (
 	"net/http"
 	"strconv"
 	"strings"
-	"sync"
 	"time"
 )
 
 var m *melody.Melody
-var stats = map[string]int{}
-var statsLock = sync.RWMutex{}
+
+const maxParticipants = 10000
+const maxMessageLength = 200
+
+// TODO: Use stats
```

Comment:
We don't need this todo, stats are now inferred from wsHub.go

Reply:
Ah, that makes sense. I'll remove it :+1: